client/reader/discover/controller.js

```diff
@@ -12,10 +11,9 @@ const ANALYTICS_PAGE_TITLE = 'Reader';
 
 const exported = {
 	discover( context, next ) {
-		const blogId = config( 'discover_blog_id' );
```

Comment:
we can remove items having to do with the blogId since this is no longer a site stream of the discover site.